doc/quickstart/find_data.rst

```diff
@@ -303,3 +303,33 @@ flexible concatenation between two cubes, depending on the particular setup:
 Note that two cube concatenation is the base operation of an iterative process of reducing multiple cubes
 from multiple data segments via cube concatenation ie if there is no time-overlapping data, the
 cubes concatenation is performed in one step.
+
+.. _extra-facets-data-finder:
+
+Use of extra facets in the datafinder
+=====================================
+Extra facets can be used to locate data files within the datafinder
```

Comment:
Could you add a link here to the extra facets section in the configuration chapter?